docs/index.md

```diff
@@ -142,6 +148,32 @@ The URL at `<<iss value from Signed JWT>>` SHALL NOT include a trailing `/`. For
 }
 ```
 
+### Certificates
+
+X.509 certificates can be used by issuers to indicate the issuer's participation in a PKI-based trust framework.
+
+If the Verifier supports PKI-based trust frameworks and the Health Card issuer includes the `"x5c"` parameter in matching JWK entries from the `.keys[]` array,
+the Verifier establishes that the issuer is trusted as follows:
+1. Verifier validates the leaf certificate's binding to the Health Card issuer by:
+    * matching the `<<iss value from Signed JWT>>` to the value
```

Comment:
Should we ask the verifier to check that the keys in the cert match the "bare key" in the JWK (or, equivalently, ask them to check that the keys in the cert have a thumprint that matches the `kid` from the health card)?

In theory, issuers are violating the JWK spec if these *don't* match; and verifiers who only care about a trust framework shouldn't be able to get in trouble even if there's a mismatch, because the PKI is all internally consistent, no matter what the bare keys contain... so I convinced myself these checks aren't required by verifiers at runtime, but I wanted to raise the question here in case this analysis is missing something.

Reply:
+1 to your conclusion that a check by the verifier isn't required for the reasons you've stated, though I think I will add a sentence reinforcing that if the issuer includes a cert, it has to match the key in the JWK entry, unless there is an objection.

Reply:
Maybe I'm missing something, but having a rogue issuer use non-matching keys yet include a x5c that chains to a trusted root is a great way to issue fake creds. I'd therefore say it's important for verifiers to check the JWK key matches the one in the cert.

Reply:
I agree with Christian, I think we should call out that while it is REQUIRED that the keys in jwks.json match to the keys in the cert, a verifier is wise to VERIFY that the do.

Reply:
> yet include a x5c that chains to a trusted root is a great way to issue fake creds

Walk me through this -- if the keys in the x5c don't match *the keys used in the SMART Health Card*, then the health card validation fails. The cert chain needs to be valid, for sure -- a verifier needs to check the chain of signatures up to the trusted root. The question I wanted to raise in this thread was: what if the keys in the leaf cert don't match the bare keys in the JWK. That's a scenario where the keys in the leaf cert have still been "blessed" by a root CA and have still been used to correctly sign a Health Card, right?

Reply:
The threat Christian is describing is only possible if the Verifier used the (crv,x,y) parameters in the JWK entry to validate the Health Card signature, and then used the X.509 certificate to validate trust. The PR text requires the Verifier to use the public key from the certificate to validate the signature specifically to prevent this.

Reply:
To Josh's question, yes, so long as the public key from the certificate is used to verify the signature, such a HC would still be valid and trusted.

Reply:
Okay, so providing a clear and safe algorithm for verifiers is critical; this algorithm can be safe even without checking for bare key :: key-in-cert equivalence, but the details definitely matter!